sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeExtensions.cs

```diff
@@ -326,5 +327,91 @@ internal static BlobAccessPolicy ToBlobAccessPolicy(this DataLakeAccessPolicy da
                 Permissions = dataLakeAccessPolicy.Permissions
             };
         }
+
+        internal static SetAccessControlRecursiveResponse DeserializeSetAccessControlRecursiveResponse(this JsonElement element)
```

Comment:
Nit - maybe add a comment warning people that this was generated to avoid tweaking arbitrarily?

Reply:
And also maybe pull this into its own file?  We're going to be updating that generator code quite often and you'll probably have less merge pain if this lives all by itself.

Reply:
Sounds good. Will do.